importers/http.go

```diff
@@ -0,0 +1,52 @@
+package importers
+
+import (
+	"net/http"
+	"net/url"
+
+	"github.com/missionMeteora/toolkit/errors"
+)
+
+// ErrEmptyLoc is returned when an empty location is provided when calling a new http importer
+const ErrEmptyLoc = errors.Error("empty importer location")
+
+// NewHTTPImporter will return a new http importer
+// Note: This is intended to be used for Slave DB's
+func NewHTTPImporter(loc string) (hp *HTTPImporter, err error) {
+	var h HTTPImporter
+	if loc == "" {
+		err = ErrEmptyLoc
+		return
+	}
+	if h.url, err = url.Parse(loc); err != nil {
+		return
+	}
+
+	hp = &h
+	return
+}
+
+// HTTP is an http importer to be used for Slave db's
+type HTTP struct {
+	hc  http.Client
+	url *url.URL
+}
+
+// SetJar will set a cookie jar for an HTTP importer
+func (h *HTTP) SetJar(jar http.CookieJar) {
```

Comment:
Cookies seems very universal for all systems. I can totally add a header option though too!